deployment/src/strongmind_deployment/ecs.py

```diff
@@ -10,25 +11,34 @@
 from pulumi_awsx.awsx import DefaultRoleWithPolicyArgs
 from strongmind_deployment import vpc
 
+class CpuArchitecture(str, Enum):
+    ARM64 = "ARM64"
+    X86_64 = "X86_64"
+
+    def __str__(self):
+        return self.value
+
+ 
 
 class EcsComponentArgs:
```

Comment:
I agree with a comment @markng  made in a code review where it is redundant to call things "Component".  

I have left this as is because the Identity service is using this right now, but I still expect some changes (and can make big changes like this easily still). 

To that end,  what is a good name for this? 

- [ ] `Ecs`
- [ ] `Container`

What makes this Layer 2 Component Special? 

*  It provides a log group (arguably shouldn't be in here)
* basic policies (eventually we will expose ability to add more policies to the roles).

The goal is that this component could be used in a `service_type` like `WebContainer` or `QueueProcessingContainer` etc.   

Points to consider in naming it.